package-res/reportviewer/reportviewer-prompt.js

```diff
@@ -284,13 +247,6 @@ define([
           me.hideGlassPane();
         }
 
-        if (me.clicking) {
-          // If "Upgrading" a Change to a Submit we do not want to process the next Submit Click, if any
-          me.ignoreNextClickSubmit = (promptMode === 'USERINPUT');
```

Comment:
We need to be very careful with this. we have many problems with this in the past i just want to make sure we dont face those issues again. On the same level, we have upgraded the lifecycle, and many of those issues might be over. Lets try it